### PR TITLE
Feature/update stackbar on zoom

### PR DESCRIPTION
**Pull request by cuginoAle**
When zooming in/out the stack-bar updates itself to reflect the nodes data actually displayed.
`Selection-controls` gets updated as well

### CODE REVIEW

File: visualizer/data-tree.js [hunk 1]
```diff
@@ -48,9 +48,16 @@ class DataTree {
     return false
   }
 
-  sortFramesByHottest () {
-    // Flattened tree, sorted hottest first, excluding the 'all stacks' root node
-    this.flatByHottest = this.getFlattenedSorted(this.getStackTopSorter())
+  sortFramesByHottest (customRootNode) {
```
Comment (goto-bus-stop) [inline visualizer/data-tree.js]: nice approach! i was expecting a `.filter()` call on flatByHottest, but i like this better :+1:

File: visualizer/stack-bar.css [hunk 2]
```diff
@@ -2,6 +2,7 @@
   position: relative;
   cursor: pointer;
   box-shadow: 0px -1px 0px rgba(255, 255, 255, 0.3) inset;
+  overflow: hidden;
```
Comment (AlanSl) [inline visualizer/stack-bar.css]: What's this for? Maybe I've missed something but I can't see what it adds

Comment (cuginoAle) [reply visualizer/stack-bar.css]: @AlanSl This prevents vertical scrollbar from appearing when the user selects the last frame on the stack-bar. If the last frame is very close to the end of the stack-bar the pointer can overflow on the right triggering scrollbars.
Also resizing the window can trigger the same behaviour as the `reDrawStackBar` is debounced and can be larger than the viewport width for a fraction of time

**Comment by AlanSl** [comment]
LGTM

**Pull request merged**